data/curated/new_submission/cleaning.R

```diff
@@ -0,0 +1,16 @@
+
+## Diabetes data from medicaldata package by Peter Higgins
+
+# Install medicaldata package
+#remotes::install_github("higgi13425/medicaldata")
+library(medicaldata)
+library(tidyverse)
+
+# Load diabetes data
+diabetes = medicaldata::diabetes
+
+# Save
+write_csv(diabetes, "diabetes.csv")

```

Comment:
```suggestion
```
Oops, I almost missed this one! Don't save within the script; `tt_save_dataset()` takes care of that part, and avoids making people write to a random location if they're following along with the script.